scripts/junit2issue.py

```diff
@@ -0,0 +1,194 @@
+'''
```

Comment:
boilerplate, copyright and license please.

Reply:
ok